src/index.ts

```diff
@@ -506,6 +498,13 @@ const ex = createServer((req, res) => {
 	resp.typeLen = (type, len) => resp
 		.setHeader('Content-Type', type)
 		.setHeader('Content-Length', len)
+	resp.send = (type, body) => {
+		resp
+		.typeLen(type, body.length)
+		.log()
+		.end(body)
+		return resp
+	}
```

Comment:
Do not change formatting for just this one method. Also, shorten it as [`ServerResponse.end()`](https://nodejs.org/api/http.html#responseenddata-encoding-callback) already returns `this`.
```suggestion
	resp.send = (type, body) => resp
		.typeLen(type, body.length)
		.log().end(body)
```